.github/workflows/pr-title-check.yml

```
@@ -8,6 +8,10 @@ on:
       - 'release/**'
       - 'hotfix/**'
     types: [opened, edited, synchronize]
+  merge_group:
+    branches:
+      - dev
+ 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)